package.json

```
@@ -78,7 +78,8 @@
     "type": "tsc --noEmit",
     "build": "etsc",
     "watch": "nodemon",
-    "test": "jest"
+    "test": "jest",
+    "prepare": "etsc"
```

```suggestion
    "prepare": "build"
```

This way, if we'd change in the future the build step for the project, `prepare` would be updated as well :)